src/tinypeel/tinypeel.py

```diff
@@ -12,19 +12,23 @@
 from itertools import repeat
 import argparse
 
-
 def runPeelingCycles(pedigree, peelingInfo, args, singleLocusMode=False):
     # Right now maf _only_ uses the penetrance so can be estimated once.
-    if args.est_alt_allele_prob:
-        PeelingUpdates.updateMaf(pedigree, peelingInfo)
     if args.alt_allele_prob_file is not None:
-        pedigree.readInAAP(pedigree, args.alt_allele_prob_file)
         for ind in pedigree:
-            if ind.MetaFounder is not None:
+            if ind.isFounder() and ind.MetaFounder is not None:
                 mfx = ind.MetaFounder
                 aaf = pedigree.AAP[mfx]
                 aafGeno = ProbMath.getGenotypesFromMaf(aaf)
                 peelingInfo.anterior[ind.idn, :, :] = aafGeno
+    else:
```

Comment:
There are still possibilities that the input alternative allele probabilities are incomplete, that is, some of the metafounders are mentioned in the pedigree file while not being specified in the alternative allele probability input file. These metafounders won't have the default alternative allele probabilities with this if-else condition.

Reply:
@RosCraddock thoughts on this comment?

Reply:
@RosCraddock do you need to add a test case for this?

Reply:
Oh, I had forgotten about this comment. I may be able to redo this to first check whether pedigree.AAP[mfx] has a value yet before assigning a default. If not, I can add an error instead. Yes, I will add a test case for this. @gregorgorjanc

Reply:
Perfect! Will leave this conversation open as a reminder to add the test case.